### DISCUSSION TITLE
Native Driver for salesforce

### DISCUSSION THREAD

**Discussion by alexsanabria** [dbeaver/dbeaver#14431] [DBeaver Development]
Hi all, I recently started working in salesforce crm and I need to make queries to the salesforce DB.
It would be great if our beloved dbeaver had a native connector for it.

thanks.

**Comment by LonwoLonwo** [comment]
You can use salesforce driver manually https://github.com/dbeaver/dbeaver/wiki/Database-drivers#adding-driver-configuration-in-dbeaver

What type of connector do you really want? Can you provide more details, please?

**Comment by LonwoLonwo** [comment]
We added Salesforce driver support.
It will be available in DBeavere 21.3.2.
